### PR TITLE
Add E2E test for pipeline check in beta

### PR DESCRIPTION
**Pull request by codyborn**
Adding an E2E test to ensure the services are working together before rolling out to production.

To make this test reliable I:
- Force open orders (don't need to worry about empty quotes)
- Decrease the amount prior to order so that the order is valid but wouldn't be filled

Before merging we need to add the following environment variables to Beta secrets
```
URA_SERVICE_URL="https://..."
GPA_SERVICE_URL="https://..."
COSIGNER_ADDRESS="..."
```

### CODE REVIEW

File: test/e2e/order.test.ts [hunk 1]
```diff
@@ -305,7 +433,7 @@ describe.skip('/dutch-auction/order', () => {
     return receipt.transactionHash
   }
 
-  describe('orders endpoint sanity checks', () => {
+  describe.skip('orders endpoint sanity checks', () => {
```
Comment (ConjunctiveNormalForm) [inline test/e2e/order.test.ts:436]: why do we skip those tests now?

File: test/e2e/order.test.ts [hunk 2]
```diff
@@ -354,34 +482,39 @@ describe.skip('/dutch-auction/order', () => {
   })
 
   describe('order endpoint sanity checks', () => {
-    it('2xx', async () => {
-      await buildAndSubmitOrder(aliceAddress, amount, DEFAULT_DEADLINE_SECONDS, WETH_GOERLI, UNI_GOERLI)
+    it.skip('2xx', async () => {
+      await buildAndSubmitOrder(aliceAddress, amount, DEFAULT_DEADLINE_SECONDS, wethAddress, uniAddress)
+    })
+
+    it('2xx with an order from URA', async () => {
```
Comment (ConjunctiveNormalForm) [inline test/e2e/order.test.ts:489]: nit: maybe move this closer to top of file? took me a while to find the case that isn't skipped haha

**Review by ConjunctiveNormalForm — APPROVED** [review]

File: test/e2e/order.test.ts (hunk 1 above)
Comment (codyborn) [reply test/e2e/order.test.ts:436]: Previously the entire block was skipped with line 43:
`describe.skip('/dutch-auction/order'`
so now that I'm not skipping it, I need to skip the individual test cases

File: test/e2e/order.test.ts (hunk 2 above)
Comment (codyborn) [reply test/e2e/order.test.ts:489]: good point

**Pull request merged**